app/src/main/java/tech/relaycorp/letro/contacts/storage/repository/ContactsRepository.kt

```diff
@@ -86,10 +89,18 @@ class ContactsRepositoryImpl @Inject constructor(
 
         if (existingContact == null || existingContact.status <= ContactPairingStatus.REQUEST_SENT) {
             if (contact.isPrivateEndpoint) {
+                val account = accountRepository.getByVeraidId(contact.ownerVeraId)!!
+                val memberIdBundle = MemberIdBundle.deserialise(account.veraidMemberBundle!!)
```

Comment:
Don't quite like to do `!!` here, but I guess this is what we have to do with the DB table we have. We probably should've created a separate table for pending account requests.

Reply:
You can write something like that to prevent possible `NullPointerException`'s
```
!! -> ?: run {
    // Log error or display notification
    return
}
```

Reply:
Good idea! Will do that 👍🏾